Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -12027,6 +12027,13 @@ - (BOOL)_allowAnimationControls
 }
 #endif
 
+- (void)dismissFormAccessoryView
+{
+    if (auto dateInput = self.dateTimeInputControl; [dateInput dismissWithAnimation])
```

Comment:
Just to confirm, `dismissWithAnimation` eventually results in `accessoryDone` right?

Reply:
Indeed — `-dismissWithAnimation` exercises logic that dismisses the popover controller programmatically, which (after this fix) will result in a call to `-accessoryDone` after the animation completes.